<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>711</width>
    <height>498</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="btn_buscar">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_exportar">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Exportar a PDF</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="table">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>691</width>
     <height>361</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>Id</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Carnet de Identidad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Nombre</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Edad</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Sexo</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Rol</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Ocupacion</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>40</y>
     <width>201</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Representantes mayores de</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="txt_buscar">
   <property name="geometry">
    <rect>
     <x>110</x>
     <y>10</y>
     <width>42</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
